code/gradle/context/dependencies.gradle

```diff
@@ -0,0 +1,70 @@
+ext.dependencyVersion = [
+        'kotlin'             : '2.0.20',
+        'google-api-client'  : '2.7.0',
+        'ejmask'             : '1.0.3',
+        'jackson'            : '2.18.1',
+        'apollo-kotlin'      : '4.0.0',
+        'apollo-adapters'    : '0.0.4',
+        'apollo-java'        : '0.0.2',
+        'apache-tika'        : '2.9.2',
+        'jakarta-validation' : '2.0.2',
+        'hibernate-validator': '6.2.5.Final',
+        'javax-validation'   : '2.0.1.Final',
+        'slf4j'              : '2.0.13',
+        'atomicfu'           : '0.24.0',
+        'dokka'              : '1.9.20'
+]
+
+ext.testDependencyVersion = [
+        'junit5': '5.10.3',
+        'assertj': '3.26.3',
+        'mockk': '1.13.13'
+]
+
```

Comment:
I'm not sure If I understand the motivation of this change, can you elaborate more on it? 
Also, doesn't this change affect renovate and dependabot updates?

Reply:
I believe renovate can be configured to handle this change. However if it comes and we face issue with the bot config we can simply move these to the main `build.gradle` file.

The motivation behind it is to enhance readability and organization of gradle configurations. I like this new way of placing deps in a separate file.

Reply:
My question is not about the new file. But structuring the versions this way looks strange for me. Is there a specific reason? I mean, we're not reusing anything here, is it just cosmetic?

Also, this could be problematic for renovate, we shouldn't add extra configs and overhead without a clear advantage